### PR TITLE
Feat/liking

### PR DESCRIPTION
**Pull request by joordy**
Fixed liking, issue is fixed!

### CODE REVIEW

File: routes/route.js [hunk 1]
```diff
@@ -68,14 +69,13 @@ async function createAccount(req, res, next) {
...
-      'email': email,
-      'password': password,
-      'gender': gender,
-      'age': age,
+      firstName: firstName,
+      lastName: lastName,
+      email: email,
+      password: password,
+      gender: gender,
+      age: age,
```
Comment (veerleprins) [inline routes/route.js:78]: This code needs back to original (for Rowan), but he already fixed it so it's good! 👍

File: routes/route.js [hunk 2]
```diff
@@ -132,26 +132,20 @@ async function home(req, res, next) {
...
-    }).toArray();
+    let allUsers = await usersCollection
+      .find({
+        $and: [
+          { id: { $ne: idLoggedIn } },
+          { id: { $nin: liked } },
+          { id: { $nin: disliked } },
+        ],
+      })
+      .toArray();
```
Comment (veerleprins) [inline routes/route.js:143]: Looks a lot better (how the enters and spaces are) 😄

**Comment by veerleprins** [comment]
Awesome that you've fixed your liking with my filter feature! 😸

File: routes/route.js [hunk 3]
```diff
@@ -89,9 +89,9 @@ async function createAccount(req, res, next) {
...
   // Rowan
   try {
-    //Veerle: Rowan, hierin moet een session beginnen met de 
+    //Veerle: Rowan, hierin moet een session beginnen met de
     //globale: idLoggedIn. < dit is de ingelogde gebruiker.
-    //Voor nu zet ik er even static code in zodat mijn code alvast kan werken: 
+    //Voor nu zet ik er even static code in zodat mijn code alvast kan werken:
     // req.session.gender = 'everyone';
     // req.session.movie = '';
     // code
```
Comment (rowanhorn1412) [inline routes/route.js:97]: you still using these comments?

**Pull request merged**